From: Chanwoo Choi <cw00.choi@samsung.com>
Date: Fri, 5 Aug 2016 18:15:46 +0900
Subject: extcon: Add new EXTCON_CHG_WPT for Wireless Power Transfer device
Git-commit: 7fe95fb889faf07ee438384858d38b820973397e
Patch-mainline: v4.9-rc1
References: fate#322003

This patchs add the new EXTCON_CHG_WPT for Wireless Power Transfer[1].
The Wireless Power Transfer is the transmission of electronical energy
from a power source. The EXTCON_CHG_WPT has the EXTCON_TYPE_CHG.

[1] https://en.wikipedia.org/wiki/Wireless_power_transfer

Signed-off-by: Chanwoo Choi <cw00.choi@samsung.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/extcon/extcon.c | 5 +++++
 include/linux/extcon.h  | 1 +
 2 files changed, 6 insertions(+)

diff --git a/drivers/extcon/extcon.c b/drivers/extcon/extcon.c
index 10ba876..7829846 100644
--- a/drivers/extcon/extcon.c
+++ b/drivers/extcon/extcon.c
@@ -93,6 +93,11 @@ struct __extcon_info {
 		.id = EXTCON_CHG_USB_SLOW,
 		.name = "SLOW-CHARGER",
 	},
+	[EXTCON_CHG_WPT] = {
+		.type = EXTCON_TYPE_CHG,
+		.id = EXTCON_CHG_WPT,
+		.name = "WPT",
+	},
 
 	/* Jack external connector */
 	[EXTCON_JACK_MICROPHONE] = {
diff --git a/include/linux/extcon.h b/include/linux/extcon.h
index e79b644..461abee 100644
--- a/include/linux/extcon.h
+++ b/include/linux/extcon.h
@@ -53,6 +53,7 @@
 #define EXTCON_CHG_USB_ACA	8	/* Accessory Charger Adapter */
 #define EXTCON_CHG_USB_FAST	9
 #define EXTCON_CHG_USB_SLOW	10
+#define EXTCON_CHG_WPT		11	/* Wireless Power Transfer */
 
 /* Jack external connector */
 #define EXTCON_JACK_MICROPHONE	20
-- 
2.10.0

